src/circle.lisp

```diff
@@ -9,20 +9,29 @@
 
 (definitializer-float circle radius)
 
-(defun make-circle (x y radius color &key (filled t))
+(defun make-circle (x y radius color &key (color2 nil) (filled t))
   (make-instance 'circle
                  :pos (make-vector2 x y)
                  :radius radius
                  :color color
+                 :color2 color2
                  :filled filled))
 
 (defmethod draw-object ((obj circle))
-  (if (filled obj)
-      (claylib/ll:draw-circle-v (c-struct (pos obj))
+  (cond
+    ((and (filled obj) (color2 obj))
+     (claylib/ll:draw-circle-gradient (truncate (x obj))
```

Comment:
Huh, so apparently we can draw a circle with gradients, or we can draw a circle with fine-tuned float positioning, but not at the same time. This is why I complain about inconsistency in Raylib's API. :-(

Reply:
Yes, I noticed your note about that applying to DRAW-CIRCLE-LINES as well. Too bad for now...